docs/conf/image.config.js

```diff
@@ -0,0 +1,389 @@
+module.exports = { // eslint-disable-line import/no-commonjs
+    "3d-buildings": {
```

Comment:
If the sizes are always the same and the basename is always just the name of the file in `docs/img/src` is it possible to just iterate over that folder and generate this list dynamically to avoid having to maintain a huge, ever growing list of image names?

Reply:
That's smart! I'll rework this so that the image config is built automatically 🙇‍♀️